airbyte-integrations/connectors/destination-selectdb/src/main/java/io/airbyte/integrations/destination/selectdb/SelectdbCopyInto.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.selectdb;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+import io.airbyte.integrations.destination.selectdb.exception.CopyIntoException;
+import io.airbyte.integrations.destination.selectdb.exception.UploadException;
+import io.airbyte.integrations.destination.selectdb.http.HttpPostBuilder;
+import io.airbyte.integrations.destination.selectdb.http.HttpPutBuilder;
+import io.airbyte.integrations.destination.selectdb.utils.ResponseUtils;
+import io.airbyte.integrations.destination.selectdb.utils.StringUtils;
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.entity.InputStreamEntity;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.concurrent.Future;
+
+public class SelectdbCopyInto {
+
```

Comment:
a lot of unused fields in this class i.e labelInfo, jdbcUrl, pendingLoadFuture, etc...

Reply:
done